src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -420,24 +420,22 @@ protected Task TestSmartTagGlyphTagsAsync(string initialMarkup, ImmutableArray<s
             string expectedMarkup,
             TestParameters parameters)
         {
-            MarkupTestFile.GetSpans(
```

Comment:
The problem here was calling GetSpans here assumed this was a single file and not the potential XML form. Since the TestWorkspace we create _will_ end up calling MarkupTestFile.GetSpans, the solution was to just delete this call, make the test workspace right away, and use the result.